test/issuehunter.js

```diff
@@ -1102,4 +1102,26 @@ contract('Issuehunter', function (accounts) {
       })
     })
   })
+
+  describe('is Mortal', function () {
+    it('does not allow to call kill', function () {
+      return Issuehunter.new().then(function (instance) {
```

Comment:
I generally use the deployed instance, `instance`, what is the difference in using `Issuehunter.new()`?